docs/index.md

```diff
@@ -11,8 +11,7 @@ hide:
 HPC-Containers Slack channel: [Join Link](http://bit.ly/hpccslack)
 
 CANOPIE HPC Workshop @ SC23
-
-TBD
+Monday morning, November 13
```

Comment:
We could state the hours as reported in the SC23 website: 8:30 am–noon

Ref: https://sc23.supercomputing.org/program/workshops/ (On-site section)